backend/dvi/mdvi-lib/fontmap.c

```diff
@@ -645,7 +645,8 @@ void	mdvi_install_fontmap(DviFontMapEnt *head)
 	}
 }
 
-static void init_static_encoding()
+static void
+init_static_encoding (void)
```

Comment:
this change breaks the style of the whole file, I think you can do the job in one line

Reply:
I agree, because it's an external code. http://mdvi.sourceforge.net/